### DISCUSSION TITLE
1.4.1 clarification on Link/button/control with surrounding text

### DISCUSSION THREAD

**Discussion by Accessibility-SME** [w3c/wcag#3126] [Q&A]
If the link/button/control (without non-color visual indicator) and the surrounding text have at least 3:1 luminance contrast, is it pass of 1.4.1?

Do we need to have hover state with non-color visual indicator?

**Comment by JAWS-test** [comment]
> If the link/button/control (without non-color visual indicator) and the surrounding text have at least 3:1 luminance contrast, is it pass of 1.4.1?

According to https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html it would pass. However, it would not be recommended because regardless of WCAG, these elements would be difficult to recognize as controls. This is especially true for visually impaired people, for whom the 3:1 contrast ratio may not be sufficient for certain colors

> Do we need to have hover state with non-color visual indicator?

No. WCAG does not require hover effects. It would be more important that the elements are easily recognizable as controls even without hover. However, hover effects are additionally useful, so that their use is recommended.

**Reply by Accessibility-SME** [reply]
Thank you @JAWS-test.

I see that in below sufficient technic it is mentioned that “Check that hovering over the link causes a visual enhancement (such as an underline, font change, etc.)” in procedure step. Is it good to have and not a requirement of 1.4.1? 

https://www.w3.org/WAI/WCAG21/Techniques/general/G183

Procedure

For each instance where color is used to convey information about text:

1. Check that the [relative luminance](https://www.w3.org/TR/WCAG21/#dfn-relative-luminance) of the color of the text differs from the relative luminance of the surrounding text by a contrast ratio of at least 3:1.

2. Check that hovering over the link causes a visual enhancement (such as an underline, font change, etc.)

**Reply by JAWS-test** [reply]
"This technique goes beyond the success criterion and asks for visual highlights when the user hovers over each link" (https://www.w3.org/WAI/WCAG21/Techniques/general/G183)

**Reply by JAWS-test** [reply]
See also https://github.com/w3c/wcag/issues/2310, https://github.com/w3c/wcag/issues/871 and https://github.com/w3c/wcag/issues/1272 for G183 and https://github.com/w3c/wcag/issues/899 for hover

**Reply by Accessibility-SME** [reply]
In below situation do we need to verify color contrast between text and button text as well under 1.4.1? 


![image](https://user-images.githubusercontent.com/127997531/229846365-976ddcf1-50e2-4edf-aa46-6902c99ac418.jpeg)

**Reply by JAWS-test** [reply]
I think not. You always have to imagine: Would the page be usable if text and buttons were the same color for everyone. The answer would be: yes, because the buttons are recognizable as buttons due to their position and labeling. But some people would certainly have problems to identify the buttons clearly as controls. Therefore the recommendation would be to design the buttons better, e.g. with a black border.

1.4.1 and 1.4.11 require sufficient contrast (3:1) only when the color is essential to recognize the elements, e.g. when links are in running text.

**Reply by Accessibility-SME** [reply]
Thank you
